src/modules/Pagination/Pagination.scss

```diff
@@ -0,0 +1,63 @@
+@import 'index.scss';
+
+.j-pagination {
+  .Pagimagic-nav {
+    &-item {
+    	display: inline-block;
+      vertical-align: top;
+      height: 30px;
```

Comment:
the number seems kind a arbitrary, also maybe this can be solve in another way that doesn't involve overriding the line height?